### PR TITLE
ISS-20: Changed Medical Condition widget to be latestObsforconceptlist

### PR DESCRIPTION
**Pull request by reagan-meant**
Issue: https://app.asana.com/0/1164864596604507/1183651948474373

### CODE REVIEW

File: omod/src/main/resources/apps/iss_patient_dashboard_app.json [hunk 1]
```diff
@@ -28,9 +28,10 @@
     "description": "Patient Medical Conditions",
     "order": 1,
     "config": {
-      "widget": "obsacrossencounters",
+      "widget": "latestobsforconceptlist",
       "icon": "icon-user-md",
       "label": "MEDICAL CONDITIONS",
+      "nLatestObs": 5,
```
Comment (ssmusoke) [inline omod/src/main/resources/apps/iss_patient_dashboard_app.json]: @reagan-meant increase this to 10, so that all conditions can be returned

Comment (reagan-meant) [reply omod/src/main/resources/apps/iss_patient_dashboard_app.json]: Done

**Comment by ssmusoke** [comment]
@reagan-meant the Intake form is broken this is what I see in Firefox developer edition 
Screen Shot 2020-08-11 at 15 13 03

**Comment by reagan-meant** [comment]
Fixed [here](https://github.com/METS-Programme/openmrs-module-ugandaemr-iss/pull/34)

**Pull request merged**